runtime/queries/rust/highlights.scm

```diff
@@ -3,9 +3,10 @@
 
 ; Assume all-caps names are constants
 ((identifier) @constant
- (#match? @constant "^[A-Z][A-Z\\d_]+$'"))
+ (#match? @constant "^[A-Z](_|[A-Z])+$"))
```

Comment:
The original query was correct, this breaks constants like `ONE1 = 1`

Reply:
The original query didn't work for me, `^[A-Z](_|[A-Z]|[0-9])+$` should fix that case though.

Reply:
Oh, I see. There was a \` there in the original query that messed it up.